src/formatter.ts

```diff
@@ -693,10 +693,13 @@ export class Formatter {
       firstContext.getMetrics().totalLeftPx;
     let targetWidth = adjustedJustifyWidth;
     let actualWidth = shiftToIdealDistances(calculateIdealDistances(targetWidth));
-    const maxX = adjustedJustifyWidth + lastContext.getMetrics().notePx;
+    const musicFont = Flow.DEFAULT_FONT_STACK[0];
```

Comment:
Not related to this PR, but lint error. Can you remove the `import { Vex } ...` from the top. (`Vex` is no longer used here.)